src/internet_identity/src/assets.rs

```diff
@@ -88,57 +88,57 @@ pub fn init_assets() {
 
 // Get all the assets. Duplicated assets like index.html are shared and generally all assets are
 // prepared only once (like injecting the canister ID).
-fn get_assets() -> [(&'static str, &'static [u8], ContentEncoding, ContentType); 8] {
-    let index_html: &[u8] = INDEX_HTML_STR.as_bytes();
-    let about_html: &[u8] = ABOUT_HTML_STR.as_bytes();
+fn get_assets() -> [(String, Vec<u8>, ContentEncoding, ContentType); 8] {
```

Comment:
The dynamic part is referring to the fact that `ASSETS` now holds bytes that can change over time rather than having only `&'static [u8]` references. This change is not concerned with how these bytes are created.
I'll update the name to `get_static_assets` in a later PR.